src/components/TaskRunnerPane/TaskRunnerPane.js

```diff
@@ -46,6 +46,7 @@ class TaskRunnerPane extends Component<Props, State> {
   handleToggleTask = taskId => {
     const { tasks, runTask, abortTask } = this.props;
 
+    // eslint-disable-next-line no-shadow
     const task = tasks.find(task => task.id === taskId);
```

Comment:
Yeah, I wish there was a way for `no-shadow` to be OK with 1-liners like this.

Oh well, that's why `eslint-disable-next-line` exists!

Reply:
I had the same reaction :)

I guess it should be possible to create a custom eslint rule based on `no-shadow` to handle this case but I'm too lazy 😄